fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -227,16 +227,23 @@ def execute(self, context):
 			isCustomImport = context.scene.ootDLImportUseCustomPath
 			scale = context.scene.ootActorBlenderScale
 			basePath = bpy.path.abspath(context.scene.ootDecompPath)
+			importAll = context.scene.ootDLImportAll
 			removeDoubles = context.scene.ootDLRemoveDoubles
 			importNormals = context.scene.ootDLImportNormals
 			drawLayer = bpy.context.scene.ootDLImportDrawLayer
 
 			filepaths = [ootGetObjectPath(isCustomImport, importPath, folderName)]
 			if not isCustomImport:
+				if importAll:
+					raise PluginError("Import All DLs requires custom import path!")
 				filepaths.append(os.path.join(bpy.context.scene.ootDecompPath, "assets/objects/gameplay_keep/gameplay_keep.c"))
 
-			importMeshC(filepaths, name, scale, removeDoubles, importNormals, drawLayer,
-				OOTF3DContext(F3D("F3DEX2/LX2", False), [name], basePath))
+			data = getImportData(filepaths)
+			dlNames = findAllDLs(data) if importAll else [name]
+			for n in dlNames:
+				print('Importing DL ' + n)
+				importMeshC(data, n, scale, removeDoubles, importNormals, drawLayer,
+					OOTF3DContext(F3D("F3DEX2/LX2", False), [n], basePath), importAll)
```

Comment:
```suggestion
					OOTF3DContext(F3D("F3DEX2/LX2", False), [n], basePath), ignoreCalls)
```
with `ignoreCalls = importAll` somewhere, would be less confusing imo

Reply:
I agree, however I'll try to do the hierarchical importing instead, so there won't be a need for ignoreCalls at all.